packages/Tag/src/tag.scss

```diff
@@ -128,13 +129,17 @@
         --ids-tag-background-primary: var(--hop-primary-surface-strong);
         --ids-tag-color-primary: var(--hop-primary-text-strong);
 
+        /* Inactive */
```

Comment:
I think we should at least put something for old brand. Even if it's the default tag look... or pick similar colors to inactive. because right now it looks like this:
![image](https://github.com/gsoft-inc/ov-igloo-ui/assets/110498164/4528e78e-15da-4785-b5dc-8cd715b83633)